Source/WebKit/DerivedSources.make

```diff
@@ -455,7 +455,12 @@ all : $(WEB_PREFERENCES_FILES)
 $(WEB_PREFERENCES_PATTERNS) : $(WTF_BUILD_SCRIPTS_DIR)/GeneratePreferences.rb $(WEB_PREFERENCES_TEMPLATES) $(WEB_PREFERENCES)
 	$(RUBY) $< --frontend WebKit $(addprefix --template , $(WEB_PREFERENCES_TEMPLATES)) $(WEB_PREFERENCES)
 
+all : $(BUILT_PRODUCTS_DIR)/DerivedSources/WebKit/HTTPHeaderNames.serialization.in
+$(BUILT_PRODUCTS_DIR)/DerivedSources/WebKit/HTTPHeaderNames.serialization.in : $(BUILT_PRODUCTS_DIR)/WebCore.framework/PrivateHeaders/HTTPHeaderNames.in $(WebKit2)/Scripts/create-http-header-serialization-interface.py
```

Comment:
I don't think $(BUILT_PRODUCTS_DIR)/WebCore.framework/PrivateHeaders/ is the correct place in the internal build.  use $(WebCorePrivateHeaders)